Cargo.toml

```diff
@@ -49,34 +48,32 @@ git = "https://github.com/KZen-networks/class-groups"
 tag = "v0.1.4"
 optional = true
 
-[dependencies.rocket]
-version = "0.4.2"
-default-features = false
-optional = true
-
-
-
-[patch.crates-io]
-rust-gmp = { version = "0.5.0", features = ["serde_support"], git = "https://github.com/KZen-networks/rust-gmp" }
-
 [dev-dependencies]
-criterion = "0.2"
+criterion = "0.3"
+rust-crypto = "0.2"
 hex = "0.3.2"
-rocket_contrib = {version = "0.4.2",default-features = false,features = ["json"]}
-rocket = {version = "0.4.2", default-features = false}
-reqwest = {version = "0.9.5", default-features = false}
+rocket = "0.4.2"
+rocket_contrib = "0.4.2"
```

Comment:
why were `default-features` and `features` removed here?
Their purpose were not importing `ring`: https://github.com/KZen-networks/multi-party-ecdsa/pull/76#issuecomment-527205651

Reply:
Agree to remove